xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -2953,7 +2963,8 @@ void CVideoPlayer::HandleMessages()
             m_ChannelEntryTimeOut.SetInfinite();
             CloseDemuxer();
             m_playSpeed = DVD_PLAYSPEED_NORMAL;
-            m_newPlaySpeed = DVD_PLAYSPEED_NORMAL;
+            m_processInfo->SetSpeed(1.0);
+            m_processInfo->SetTempo(1.0);
```

Comment:
Not sure if it's worth, but these two lines repeat 3 times, maybe add a function ```SetSpeedAndTempo```?